bigquery.tf

```diff
@@ -3,6 +3,25 @@ locals {
   bigquery_dataset_name         = coalesce(var.bigquery_dataset_name, local.default_bigquery_dataset_name)
 }
 
+resource "google_bigquery_connection" "bq_spark_connection" {
+  connection_id = "bq-spark-conn-${lower(var.organisation_id)}-${lower(var.country_code)}"
+  location      = var.storage_location
+  description   = "BQ spark connection for ML"
+  spark {
+
+  }
+}
+
+resource "google_bigquery_connection_iam_binding" "binding" {
```

Comment:
Please don't use binding, because if on remote client side there would be some permissions given of this kind, it would potentially wipe them out and cause unintentional breakages.

Reply:
What we want is this I think: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_connection_iam#google_bigquery_connection_iam_member